.github/workflows/e2e.yaml

```diff
@@ -72,4 +72,8 @@ jobs:
           - "-X main.version=${{ github.sha }}"
         EOF
         docker run $(go run ./ build ./test/ --platform=${PLATFORM}) --wait=false 2>&1 | grep "${{ github.sha }}"
+        
+        # Check that using sbom-dir works.
+        KO_DOCKER_REPO=""  go run ./ build -t test --push=false --sbom-dir ./sbom-data ./test | grep "Writing SBOM to"
+        jq . ./sbom-data/ko-app/*.json 
```

Comment:
The output should not be written to a `ko-app` directory, it should be a filename like `sbom-dir/<import-path>.json` (with slashes replaced).